plugins/mcreator-localization/lang/texts.properties

```diff
@@ -1079,6 +1079,7 @@ blockly.block.entity_pos_y=Y position of %1
 blockly.block.entity_pos_z=Z position of %1
 blockly.block.entity_potioneffectlevel=Get effect level/amplifier of potion %2 for %1
 blockly.block.entity_potioneffectremaining=Get remaining ticks of potion %2 for %1
+blockly.block.entity_push=Push %4 with x: %1 y: %2 z: %3
```

Comment:
Arguments are dx dy and dz as they define push delta value

Reply:
Ok, anything else to fix?

Reply:
Done